test/registration_spec.rb

```diff
@@ -279,23 +265,16 @@
   end
 
   describe ".is_registered?" do
-    let(:destdir) { "/foo" }
-
-    it "honors the target system prefix at upgrade" do
-      expect(Yast::Mode).to receive(:update).and_return(true)
-      expect(Yast::Stage).to receive(:initial).and_return(true)
-      expect(Yast::Installation).to receive(:destdir).and_return(destdir)
-      expect(File).to receive(:exist?).with(/\A#{Regexp.escape(destdir)}/)
-      Registration::Registration.is_registered?
+    it "returns true if the global credentials file exists" do
+      expect(File).to receive(:exist?).with(SUSE::Connect::YaST::GLOBAL_CREDENTIALS_FILE)
+        .and_return(true)
+      expect(Registration::Registration.is_registered?).to eq(true)
     end
 
-    it "does not add the prefix if not at upgrade" do
-      allow(Yast::Mode).to receive(:update).and_return(false)
-      allow(Yast::Stage).to receive(:initial).and_return(false)
-      expect(Yast::Installation).to_not receive(:destdir)
+    it "returns false if the global credentials file exists" do
```

Comment:
"....if the global credential file **does not** exist", right?

Reply:
Oh, right, copy&paste error...